bin/antlr4.js

```diff
@@ -2,5 +2,11 @@
 
 var shell = require('shelljs');
 
-var arguments = ' ' + process.argv.splice(2).join(' ');
+var arguments = process.argv.splice(2);
+
+for (i=0;i<arguments.length; i++){
+  arguments[i] = arguments[i].replace(/'|"|;|&|,|\\|\|/gi, '')
+}
+
+var arguments = ' ' + arguments.join(' ');
 shell.exec('java -jar ' + __dirname + '/antlr-4.5-complete.jar' + arguments);
```

Comment:
I prefer to directly find a way to fix that function.

Reply:
Not sure why I reverted my first change :sweat_smile:, but now its properly fixed using fileExec